site/content/en/docs/Contributing/triage.md

```diff
@@ -0,0 +1,218 @@
+---
+linkTitle: "Triage"
+title: "Triaging Minikube Issues"
+date: 2020-03-17
+weight: 10
+description: >
+  How to triage issues in the minikube repo
+---
+
+Triage is an important part of maintaining the health of the minikube repo.
+A well organized repo allows maintainers to prioritize feature requests, fix bugs, and respond to users facing difficulty with the tool as quickly as possible.
+
+Triage includes:
+- Labeling issues
+- Responding to issues
+- Closing issues
+
+If you're interested in helping out with minikube triage, this doc covers the basics of doing so.
+
+Additionally, if you'd be interested in participating in our weekly triage meeting, please fill out this [form](https://forms.gle/vNtWZSWXqeYaaNbU9) to express interest. Thank you! 
+
+# Daily Triage
+Daily triage has two goals:
+
+1. Responsiveness for new issues
+1. Responsiveness when explicitly requested information was provided
+
+The list of outstanding items are at https://teaparty-tts3vkcpgq-uc.a.run.app/s/daily-triage - it covers:
```

Comment:
That's an ugly URL :( Do you mind changing this to: http://tinyurl.com/mk-tparty/daily-triage (here and elsewhere)

Reply:
for sure!